src/fsharp/FSharp.Core/Linq.fsi

```diff
@@ -109,6 +117,15 @@ namespace Microsoft.FSharp.Linq
         [<CompiledName("ToSByte")>]
         val inline sbyte      : value:Nullable< ^T > -> Nullable<sbyte>      when ^T : (static member op_Explicit : ^T -> sbyte)      and default ^T : int
         
+        
+        /// <summary>Converts the argument to signed byte. This is a direct conversion for all 
+        /// primitive numeric types. The operation requires an appropriate
+        /// static conversion method on the input type.</summary>
+        /// <param name="value">The input value.</param>
+        /// <returns>The converted sbyte</returns>
+        [<CompiledName("ToInt8")>]
```

Comment:
If new APIs are being added, you will need to update the surface area tests in FSharp.Core.Unittests.  Note that ATM this test is failing, as some required updates were missed during recent PRs. @KevinRansom I think is working on fixing it.